src/core/server/elasticsearch/elasticsearch_service.ts

```diff
@@ -164,18 +164,7 @@ export class ElasticsearchService implements CoreService<InternalElasticsearchSe
       ignoreVersionMismatch: config.ignoreVersionMismatch,
       esVersionCheckInterval: config.healthCheckDelay.asMilliseconds(),
       kibanaVersion: this.kibanaVersion,
-    }).pipe(publishReplay(1));
```

Comment:
Could you add a test that it behaves as expected? It's easy to break with the next refactoring.